starboard/media.h

```diff
@@ -630,47 +626,47 @@ typedef struct SbMediaAudioSampleInfo {
-//   solely based on the key system and the known attributes.  For example, if
-//   an implementation supports "com.widevine.alpha", it should also return
-//   `kSbMediaSupportTypeProbably` when |key_system| is
-//   |com.widevine.alpha; invalid_attribute="invalid_value"|.
-//   Currently the only attribute has to be supported is |encryptionscheme|.  It
-//   reflects the value passed to `encryptionScheme` of
-//   MediaKeySystemMediaCapability, as defined in
-//   https://wicg.github.io/encrypted-media-encryption-scheme/, which can take
-//   value "cenc", "cbcs", or "cbcs-1-9".
-//   Empty string is not a valid value for |encryptionscheme| and the
-//   implementation should return `kSbMediaSupportTypeNotSupported` when
-//   |encryptionscheme| is set to "".
-//   The implementation should return `kSbMediaSupportTypeNotSupported` for
-//   unknown values of known attributes.  For example, if an implementation
-//   supports "encryptionscheme" with value "cenc", "cbcs", or "cbcs-1-9", then
-//   it should return `kSbMediaSupportTypeProbably` when |key_system| is
-//   |com.widevine.alpha; encryptionscheme="cenc"|, and return
-//   `kSbMediaSupportTypeNotSupported` when |key_system| is
-//   |com.widevine.alpha; encryptionscheme="invalid"|.
-//   If an implementation supports key system with attributes on one key system,
-//   it has to support key system with attributes on all key systems supported.
+// |key_system|: A lowercase value in the form of "com.example.somesystem",
+// and can be matched exactly with known DRM key systems of the platform.
+// When |key_system| is an empty string, the return value is an indication for
+// non-encrypted media. See https://w3c.github.io/encrypted-media/#key-system
+//
+// An implementation may choose to support |key_system| with extra attributes,
+// separated by ';', like
+// |com.example.somesystem; attribute_name1="value1"; attribute_name2=value1|.
+// If |key_system| with attributes is not supported by an implementation, it
+// should treat |key_system| as if it contains only the key system, and reject
+// any input containing extra attributes, i.e. it can keep using its existing
+// implementation.
+// When an implementation supports |key_system| with attributes, it has to
+// support all attributes defined by the Starboard version the implementation
+// uses.
+// An implementation should ignore any unknown attributes, and make a decision
+// solely based on the key system and the known attributes. For example, if
+// an implementation supports "com.widevine.alpha", it should also return
+// |kSbMediaSupportTypeProbably| when |key_system| is
+// |com.widevine.alpha; invalid_attribute="invalid_value"|.
+// Currently the only attribute has to be supported is |encryptionscheme|. It
+// reflects the value passed to |encryptionScheme| of
+// MediaKeySystemMediaCapability.
+// This can take value "cenc", "cbcs", or "cbcs-1-9".
```

Comment:
This line should start on the previous one, so they can be in the same paragraph.  This => It.

Reply:
Fixed.